index.html

```diff
@@ -55,6 +54,227 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
     </div>
  
   </section>
+
+
+  <button class="arrow">
+    <img src="images/buttons/Disabled.png" alt="">
+  </button>
+
+  <section class="recent-work">
+    <header>
+      <h2 class="green">
+        My recent work
+      </h2>
+      <hr class="green">
+    </header>
+
+    <div class="projects">
+
+      <!-- Project 1 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Setup and mobile version skeleton
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
```

Comment:
- [ ] Kindly try to match your design to the Figma design. The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 
 
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_work](https://user-images.githubusercontent.com/70483835/227732183-ec40b9e4-e020-4651-adae-7f4b12807dd6.png) | ![figma_work](https://user-images.githubusercontent.com/70483835/227732181-9bcfe7a0-28ec-44b6-8878-f361da517230.png)